examples/Cargo.toml

```diff
@@ -7,11 +7,13 @@ description = "Examples using the Turnkey Rust SDK"
 publish = false
 
 [dependencies]
+turnkey_proofs = { path = "../proofs" }
 turnkey_api_key_stamper = { path = "../api_key_stamper" }
 turnkey_enclave_encrypt = { path = "../enclave_encrypt" }
 turnkey_client = { path = "../client" }
 dotenvy = { version = "0.15.0" }
 hex = { version = "0.4.3", default-features = false, features = ["std"] }
 reqwest = { version = "0.11", features = ["json"] }
 tokio = { version = "1.44.1", features = ["full"] }
-serde_json = { version = "1.0.140" }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0" }
```

Comment:
nit: any particular reason for this downgrade?

Reply:
No particular reason, I just copied what we had in other crates. Let me align everything on `1.0.x` where x is the latest.

Reply:
Addressed in https://github.com/tkhq/rust-sdk/pull/18/commits/80503b4fd59fa94b588f6fdcdc71081a1bdb4576